util/chunk/disk.go

```diff
@@ -47,8 +47,11 @@ var bufReaderPool = sync.Pool{
 var tmpDir = path.Join(os.TempDir(), "tidb-server-"+path.Base(os.Args[0]))
 
 func init() {
-	_ = os.RemoveAll(tmpDir) // clean the uncleared temp file during the last run.
-	err := os.Mkdir(tmpDir, 0755)
+	err := os.RemoveAll(tmpDir) // clean the uncleared temp file during the last run.
+	if err != nil {
```

Comment:
If `tmpDir` does not exist, we should not write a log here.

Reply:
> RemoveAll removes path and any children it contains. It removes everything it can but returns the first error it encounters. If the path does not exist, RemoveAll returns nil (no error). If there is an error, it will be of type *PathError.